demo/src/app/default/default.component.html

```diff
@@ -1,26 +1,36 @@
-<div class="jumbotron">
+<main class="bd-masthead">
   <div class="container">
-    <div class="pull-lg-left">
-      <h1 class="display-3">ng-bootstrap</h1>
-      <hr class="m-y-2 header">
-      <p>Angular 2, powered by Bootstrap 4 written by the angular-ui Team</p>
-    </div>
+    <img class="bd-booticon" src="img/ngb-logo.png" />
+    <p class="lead">
+      Bootstrap 4 components, powered by Angular 2 and written by the angular-ui Team.
+    </p>
+    <p class="lead">
+      <a
+        class="btn btn-outline-secondary"
+        [routerLink]="['/components']">Demo</a>
+      <a
+        class="btn btn-outline-secondary"
+        [routerLink]="['/getting-started']">Installation instructions</a>
+    </p>
+    <p>
+      Currently v{{version}}
+    </p>
   </div>
-</div>
+</main>
 
 <section class="container">
+  <h2>Welcome</h2>
   <p class="lead">
-    Welcome to the Angular 2 version of the <a target="_blank" href="https://github.com/angular-ui/bootstrap">Angular UI Bootstrap</a>
-    library. This library is being built from scratch in Typescript using the Bootstrap 4 CSS framework.
+     Here we are, the Angular 2 version of the <a target="_blank" href="https://github.com/angular-ui/bootstrap">Angular UI Bootstrap</a>
+     library. This library is being built from scratch in Typescript using the Bootstrap 4 CSS framework.
   </p>
   <p class="lead">
+    To get started quickly you can follow <a [routerLink]="['/getting-started']">installation instructions</a> or just browse <a [routerLink]="['/components']">demos</a> for the existing components.
+  </p>
+  <p class="lead text-muted">
```

Comment:
please remove the `text-muted` class.  i think this is ok to be black like the other two paragraphs.